### ISSUE TITLE
Input OTP extract entered value

### ISSUE THREAD

**Issue by proficientdesigners** [sunarya-thito/shadcn_flutter#148]
For the past three days i've been exploring this dependency for flutter mobile and the components available are good for easy usage, but when it comes to InputOTP why can't i able to get the exact value entered, instead i guess im getting array of key values.

For eg: If i enter this set of numbers `[1, 2, 3, 4, 5, 6]` im getting this as a set values from onChanged function `[49, 50, 51, 52, 53, 54]`

Am i miss understanding the Usage?

Note: Im new to flutter too, my basic reputation is from ReactJS

**Comment by sunarya-thito** [comment]
The values for InputOTP are codepoints (character), not the integer value itself.
See #51

**Comment by proficientdesigners** [comment]
Yeah, I guessed that before, but can you please mention the same in your documentation so that others will also know what to do exactly.
